base/Base.jl

```diff
@@ -121,6 +121,23 @@ function setpropertyonce!(x::Module, f::Symbol, desired, success_order::Symbol=:
     return Core.setglobalonce!(x, f, val, success_order, fail_order)
 end
 
+julia_debug::Bool = true
```

Comment:
Makes me a bit nervous to have a compiler time setting that I can easily change by writing a global.